README.md

```diff
@@ -46,19 +39,12 @@ You will need the following items to run the sample:
 3. Configure Chat Copilot.
   
     ```powershell
-    .\Configure.ps1 -AIService {AI_SERVICE} -APIKey {API_KEY} -Endpoint {AZURE_OPENAI_ENDPOINT} -ClientId {AZURE_APPLICATION_ID} 
+    .\Configure.ps1 -AIService {AI_SERVICE} -APIKey {API_KEY} -Endpoint {AZURE_OPENAI_ENDPOINT}
```

Comment:
Need to update local scripts to take client/tenant/instance as optional params

Reply:
and to set `REACT_APP_AUTH_TYPE=AzureAd` in `.env` if these params are passed